xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.559.xml

```diff
@@ -112,18 +114,37 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><persName><name type="gentilicium"><gap reason="lost" extent="unknown" unit="character"/>ae</name> <g type="hedera"/>
+              <lb n="2"/><name type="cognomen" key="Primigenia?Archigenia?Iphigenia" cert="low"><supplied reason="lost" cert="low">Prim</supplied>igeniae</name></persName>
+              <lb n="3"/><persName><name type="gentilicium"><gap reason="lost" extent="unknown" unit="character"/>ius</name>
+                 <name type="cognomen" key="Ampliatus?Amandus?Amarantus?Ambrosius?Amerimnus">Am
```

Comment:
Keying it like this is a bit weird, since that's what will display in the index; put only the things that you put in the text and put the other options in a commentary note.